src/screens/CreatePost/index.js

```diff
@@ -882,8 +861,8 @@
     justifyContent: 'flex-start',
     overflow: 'scroll'
   },
-  hastagText: {
-    color: COLORS.gray110,
+  hashtagText: {
+    color: COLORS.gray410,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `hashtagText` style has been changed from `COLORS.gray110` to `COLORS.gray410`. Ensure that this change was intentional and that it improves the readability or appearance of the text.
```diff
+ hashtagText: {
+   color: COLORS.gray410,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->